Functions/Quantification/xASL_quant_HadamardDecoding.m

```diff
@@ -213,18 +213,20 @@
 
 % INNER REPETITIONS
 % Normally, for a 4-block Hadamard, we acquire 4 volumes. But there may be reasons to repeat the readout, resulting in more volumes.
-% E.g., with a multi-TE readout, each Hadamard-encoded volume is acquired for each echo, e.g., resulting in 8 volumes per Hadamard block for an 8-echoes readout.
-% For 4 Hadamard blocks, this results in 8*4 = 32 volumes for the full sequence.
+% E.g., with a multi-TE readout, each Hadamard-encoded volume is acquired for each echo. For example, a 4-block Hadamard with 8-TEs
+% results in 8*4 = 32 volumes for the full sequence.
 % Here, we define these here as "inner repetitions".
 %
-% innerRepetition: single-volume repetitions within a Hadamard block
+% innerRepetition: single-volume repetitions for each PLD
```

Comment:
As above

Reply:
No we don't say blocks = encoded volumes anywhere. And I keep constantly changing it back in the comments.
Blocks are subboli. the number of blocks  == the number of volumes. Blocks ~= volumes - I wrote that in like 5 comments above already.

I never really said you should call it nBlocks. You have started rewriting the variable names and comments without asking, I have tried to fix things that are obviously wrong. Using nBlocks is not really incorrect. If it would be up to me, I would have used TimeEncodedMatrixSize everywhere.